components/StatusCheckBox/StatusCheckbox.tsx

```diff
@@ -0,0 +1,56 @@
+import { StatusCheckboxProps } from '@/utils/constants/leadslist-constant';
+import React from 'react';
+
+const StatusCheckbox: React.FC<StatusCheckboxProps> = ({
+  status,
+  handleCheckboxChange,
+}) => {
+  return (
+    <div className='flex ml-1 my-1'>
+      <div className='flex items-center me-4'>
```

Comment:
Not this way. I would like to suggest you to make a reusable component. It will be looks like:
```javascript
<div className='flex items-center me-4'>
  <input
    id={id}
    type='checkbox'
    checked={checked}
    onChange={onChange}
    className='w-4 h-4 text-blue-600 bg-gray-100 border-gray-300 rounded focus:ring-blue-500 dark:focus:ring-blue-600 dark:ring-offset-gray-800 focus:ring-2 dark:bg-gray-700 dark:border-gray-600'
  />
  <label
    htmlFor={id}
    className='ms-2 text-sm font-medium text-gray-900 dark:text-gray-300'>
    {children}
  </label>
</div>
```

While we will use it, it will be looks like:
```javascript
<StatusCheckbox
  id='hot-checkbox'
  checked={status.hot}
  onChange={() => handleCheckboxChange('hot')}>
  Hot
</StatusCheckbox>

```